man/strucchange.internal.Rd

```diff
@@ -18,4 +18,7 @@
 \description{
   These are not to be called by the user.
 }
+\value{
+  Various, used internally and subject to change.
```

Comment:
Wouldn't it be better to **not** export all the internal functions and then simply removing the documentation at all? Would this be difficult?

Reply:
Yes, and that's what Achim indicated in his NAMESPACE himself, but apparently he never got around to it. And since we'd like to keep source code compatibility with upstream, better not to touch it at the moment, I think.

Reply:
OK, let's try.